contrib/linux/actions/service.py

```diff
@@ -18,7 +18,7 @@
 import re
 import sys
 import os
-import platform
+import distro as platform
```

Comment:
IIRC, that won't work since that ``service.py`` file is used for remote shell runner which means we copy file to a remote host first where it's executed (and there ``distro`` package may not be available).

In short - Python script which use ``remote-shell-script`` runner can't rely on any 3rd party dependencies since they run using system Python and not using a pack virtual environment.

/cc @blag

Reply:
It seems like one, probably not as robust but probably good alternative would be to just shell out to ``lsb_release`` in this module - https://github.com/mp15/irods/commit/fd8af0c9154d2fa7308c0562c18b2c236e748511.

Reply:
This should do it - b6cb22d2f0184b621c123e94dda28a324eefb7ac.